examples/tests/sdk_tests/resource/on-stop.test.w

```diff
@@ -0,0 +1,29 @@
+bring sim;
+
+inflight class OnStopThrowerBackend impl sim.IResource {
+  pub onStart(ctx: sim.IResourceContext) {}
+  pub onStop() {
+    throw "unexpected error!";
+  }
```

Comment:
When a resource fails while stopping, there's a pretty good argument for gracefully handling the error so that other resources can also clean up afterwards and release any of their resources, so that's basically what we do.

Reply:
Agreed